src/layers/core/hexagon-layer/hexagon-layer.js

```diff
@@ -92,8 +92,16 @@ export default class HexagonLayer extends Layer {
 
     const {attributeManager} = this.state;
     attributeManager.addInstanced({
-      instancePositions: {size: 3, update: this.calculateInstancePositions},
-      instanceColors: {size: 4, update: this.calculateInstanceColors}
+      instancePositions: {
+        size: 3,
+        update: this.calculateInstancePositions
```

Comment:
HexagonLayer uses multiple accessors to generate instanceColors. `getAlpha` prop should be removed for consistency, though. @heshan0131